src/main/java/io/nats/client/Options.java

```diff
@@ -565,6 +565,14 @@ public Thread newThread(Runnable r) {
         }
     }
 
+    /**
+     * Creates a builder for the options.
+     * @return the builder.
+     */
+    public static Builder newBuilder() {

```

Comment:
1. To be consistent with all the other builders, please name this method just `builder`. 
2. Can you move it below the comments for the builder, just before the class declaration.
3. While you are there,  can you remove the comment (line 582 in this code) `is equivalent to calling {@link Nats#connect() Nats.connect()}.` This isn't right
4. Last, can you please sign your commits. https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits